apps/web/vibes/soul/components/button/index.tsx

```diff
@@ -41,7 +41,7 @@ export const Button = function Button({
         'after:absolute after:inset-0 after:z-0 after:h-full after:w-full after:rounded-full',
         'after:ease-[cubic-bezier(0.075,0.365,0.000,0.995)] after:transition-[opacity,transform] after:duration-500',
         'after:-translate-x-[110%]',
-        !loading && !disabled && 'hover:after:translate-x-0',
+        !Boolean(loading) && !Boolean(disabled) && 'hover:after:translate-x-0',
```

Comment:
Instead of coercing these values, I think we should provide default values so that they're always a boolean. Alternatively, we can do `loading !== true` or `disabled !== true`. I personally prefer having the values always have a boolean as it makes things easier to follow